runner/src/server/plugins/applicationStatus.js

```diff
@@ -107,11 +127,11 @@ const applicationStatus = {
           const { pay } = await cacheService.getState(request)
           const { meta } = pay
           meta.attempts++
-          // TODO:- let payService handle shortid.generate()
-          const reference = `FCO-${shortid.generate()}`
-          const res = await payService.payRequest(meta.amount, reference, meta.description, meta.payApiKey)
+          // TODO:- let payService handle nanoid(10)
+          const reference = `FCO-${nanoid(10)}`
```

Comment:
sorry, you're free to remove `FCO-`.. you probably don't want HO payment references be prepended with FCO heh